modules/server/src/main/scala/observe/server/ObserveEnvironment.scala

```diff
@@ -12,17 +12,17 @@ import observe.server.tcs.Tcs
  * Describes the parameters for an observation
  */
 final case class ObserveEnvironment[F[_]](
-  odb:      OdbProxy[F],
-  dhs:      DhsClient[F],
-  config:   CleanConfig,
-  stepType: StepType,
-  obsId:    Observation.Id,
-  dataId:   DataId,
-  inst:     InstrumentSystem[F],
-  insSpecs: InstrumentSpecifics,
-  otherSys: List[System[F]],
-  headers:  HeaderExtraData => List[Header[F]],
-  ctx:      HeaderExtraData
+  odb:       OdbProxy[F],
```

Comment:
Do we need an `OdbProxy`?

Reply:
Yes, but with a different implementation.